crates/driver/src/domain/competition/solution/mod.rs

```diff
@@ -32,6 +32,7 @@ pub use {interaction::Interaction, settlement::Settlement, trade::Trade};
 #[derive(Debug, Clone)]
 pub struct Solution {
     pub id: Id,
+    pub score: Option<Score>, // TODO remove Option
```

Comment:
Same here.

Reply:
I am not sure score needs to be optional at this specific place. What I was aiming with this is:
1. Between autopilot and driver, score should not be optional!
2. Between driver and solver, score should be optional. As CIP20 states, solvers can provide score discount, which means, in that case, the driver needs to calculate the score=objective_value and reduce it with the score discount. This is again, to support the transition period when we switch to colocation.